app/assets/stylesheets/conversation_show.scss

```diff
@@ -80,7 +80,7 @@ a {
 }
 
 .self .chatboxmessagecontent {
-  background-color: $grey-l;
+  background-color: #71D8CF;
```

Comment:
Color literals like `#71D8CF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#71D8CF` should be written as `#71d8cf`